podcasts/End of Year/Stories/Views/CommonStoryViews.swift

```diff
@@ -423,3 +423,49 @@ extension NSLocale {
         return currentLanguageCode.hasPrefix("en")
     }
 }
+
+// MARK: - 2023 background
+
+struct StoryGradient: View {
+    var body: some View {
+        Rectangle()
+        .foregroundColor(.clear)
+        .frame(width: 510, height: 510)
+        .background(
+            LinearGradient(
+                stops: [
+                    Gradient.Stop(color: Color(red: 0.25, green: 0.11, blue: 0.92), location: 0.00),
+                    Gradient.Stop(color: Color(red: 0.68, green: 0.89, blue: 0.86), location: 0.61),
+                    Gradient.Stop(color: Color(red: 0.87, green: 0.91, blue: 0.53), location: 1.00),
+                ],
+                startPoint: UnitPoint(x: 0.49, y: 0.11),
+                endPoint: UnitPoint(x: 0.49, y: 0.98)
+            )
+        )
+        .cornerRadius(510)
+        .blur(radius: 107)
+        .opacity(0.6)
+    }
+}
+
+struct PlusStoryGradient: View {
+    var body: some View {
+        Rectangle()
+        .foregroundColor(.clear)
+        .frame(width: 510, height: 510)
+        .background(
+            LinearGradient(
+                stops: [
+                    Gradient.Stop(color: Color(red: 0.91, green: 0.35, blue: 0.26), location: 0.00),
+                    Gradient.Stop(color: Color(red: 0.87, green: 0.91, blue: 0.53), location: 0.61),
+                    Gradient.Stop(color: .black, location: 1.00),
+                ],
+                startPoint: UnitPoint(x: 0.49, y: 0.11),
+                endPoint: UnitPoint(x: 0.49, y: 0.98)
+            )
+        )
+        .cornerRadius(510)
+        .blur(radius: 107)
+        .opacity(0.55)
+    }
+}
```

Comment:
Since these are basically the same, could we not just combine them and add a flag to determine which colors to show?